client/app/dedicated/housing/dedicated-housing.html

```diff
@@ -10,46 +10,12 @@
         <div class="row">
             <div class="col-sm-9">
                 <div class="mb-2">
-                    <h1 class="word-break mb-0"
+                    <h1 class="oui-header_2 word-break mb-0"
```

Comment:
`h1` with `h2` style?

Reply:
Yes, it seems weird, but since I want to keep headers order and apply specific style (view this previous comment : https://github.com/ovh-ux/ovh-manager-dedicated/pull/264/files/82dec106ef004325948369ee6283343c0d2bd585#diff-8150bf2374f0c553e9b7ca6e221f8392)

Reply:
Makes sense to me, still better than using `h2` without an `h1`